src/corelibs/U2Core/src/datatype/msa/MultipleChromatogramAlignmentRow.cpp

```diff
@@ -400,56 +400,31 @@ QPair<DNAChromatogram::ChromatogramTraceAndValue, DNAChromatogram::ChromatogramT
     return {peaks[0], peaks[1]};
 }
 
-bool MultipleChromatogramAlignmentRowData::isRowContentEqual(const MultipleChromatogramAlignmentRow &row) const {
-    return isRowContentEqual(*row);
+bool MultipleChromatogramAlignmentRowData::isEqual(const MultipleAlignmentRowData &other) const {
+    CHECK(other.type == MultipleAlignmentDataType::MCA, false);
+    auto mcaRow = dynamic_cast<const MultipleChromatogramAlignmentRowData *>(&other);
+    SAFE_POINT(mcaRow != nullptr, "Not an MCA row!", false);
```

Comment:
Use `L10N::nullPointer` instead

Reply:
This is not a null-ptr error, this is an invalid type error.